packages/expo-random/src/ExpoRandom.web.ts

```diff
@@ -0,0 +1,9 @@
+export default {
+  get name(): string {
+    return 'ExpoRandom';
+  },
+  async getRandomIntegerAsync(length: number): Promise<Uint8Array> {
+    const array = new Uint8Array(length);
+    return window.crypto.getRandomValues(array);
```

Comment:
The API layer is going to try to interpret this as a Base64 string. In this one case since the web API fills a Uint8Array, it'd be kind of dumb to have to encode it as Base64 and then decode it again. Plus eventually when we have Typed Arrays from the native JSC side, we can get rid of Base64 altogether (and the base64-js dependency).

So something like this might make sense in the native layer:

```js
let bytes: Uint8Array;
if (ExpoRandom.getRandomBytesAsync) {
  bytes = await ExpoRandom.getRandomBytesAsync(byteCount);
} else if (ExpoRandom.getRandomBase64StringAsync) {
  const base64 = await ExpoRandom.getRandomBase64StringAsync(byteCount);
  bytes = toByteArray(base64);
} else {
  throw new UnavailabilityError('expo-random', 'getRandomBytesAsync');
}

return bytes;
``` @